docs/docs/03-light-clients/05-tendermint/01-overview.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Overview
+sidebar_label: Overview
+sidebar_position: 1
+slug: /ibc/light-clients/tendermint/overview
+---
+
+# `07-tendermint`
+
+## Overview
+
+The tendermint client is the first and most deployed light client in IBC. It implements the IBC Light Client interface to track a counterparty running Comet BFT consensus. Note: Tendermint is the old name of Comet BFT which has been retained in IBC to avoid expensive migration costs.
```

Comment:
I think it is important to give some technical context in the overview. For example, the features it supports and how it differs from a basic header tracking light client. Mentioning support of historical proofs and misbehaviors.

Possibly mention the state the needs to be stored by the light client i.e.:

- Client State (as parameters)
- A mapping from Height -> Consensus State

Otherwise it can be difficult for the first time reader to understand client state is stored as a singleton but consensus states are stored as maps later on.